tsdb/wal/wal_test.go

```diff
@@ -204,6 +208,21 @@ func TestWALRepair_ReadingError(t *testing.T) {
 	}
 }
 
+func TestUnsequentialSegments(t *testing.T) {
+	segsN := 5 * pageSize
```

Comment:
this determines how big is the wal and not how many wal files it will create.

Reply:
This is done in order to prevent `error: invalid segment` which is due to `segsN%pageSize != 0`.